protected/components/DownloadService.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Service to provide tokens for password reset functionality
```

Comment:
The DownloadService's PHPDoc class  description seems wrong to me.

Reply:
Fixed now.